test/src/module/postgres/interfaceTest.c

```diff
@@ -72,12 +68,15 @@ testRun(void)
         //--------------------------------------------------------------------------------------------------------------------------
         storagePutP(
             storageNewWriteP(storageTest, controlFile),
-            pgControlTestToBuffer((PgControl){.version = PG_VERSION_11, .systemId = 0xFACEFACE, .walSegmentSize = 1024 * 1024}));
+            pgControlTestToBuffer(
+                (PgControl){
+                    .version = PG_VERSION_11, .systemId = 0xFACEFACE, .catalogVersion = 201809051, .walSegmentSize = 1024 * 1024}));
 
         PgControl info = {0};
         TEST_ASSIGN(info, pgControlFromFile(storageTest), "get control info v11");
         TEST_RESULT_UINT(info.systemId, 0xFACEFACE, "   check system id");
         TEST_RESULT_UINT(info.version, PG_VERSION_11, "   check version");
+        TEST_RESULT_UINT(info.catalogVersion, 201809051, "   check version");
```

Comment:
copy paste error - "check version" line 79 should be "check catalog version" (already have check version line 78)

Reply:
Fixed.